gix-ref/tests/fixtures/make_packed_ref_repository_for_overlay.sh

```diff
@@ -10,10 +10,13 @@ git branch A
 git tag -m "tag object" tag-object
 
 mkdir -p .git/refs/remotes/origin
+mkdir -p .git/refs/heads/sub/dir
 
 cp .git/refs/heads/main .git/refs/remotes/origin/
 
 echo "ref: refs/remotes/origin/main" > .git/refs/remotes/origin/HEAD
+echo "ref: refs/heads/main" > .git/refs/heads-packed
+echo "ref: refs/heads/main" > .git/refs/heads/sub/dir/packed
```

Comment:
I think these names should be changed so `packed` isn't included. This is because symbolic refs are never packed.

Reply:
I just wanted to make sure that both the loose and packed refs are handled correctly and not accidentally testing the wrong thing. Changed the names to `refs/feature-suffix` and `refs/feature/sub/dir/algo` to make the test more explicit.